src/pages/ListPage/ListPage.js

```diff
@@ -1,5 +1,167 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { useLocation, useSearchParams, useNavigate } from 'react-router-dom';
+import { BiChevronDown } from 'react-icons/bi';
+import ListMenu from './ListHeader/ListMenu/ListMenu';
+import SubMenus from './ListHeader/ListMenu/SubMenus';
+import Product from './Products/Product';
 
-export default function ListPage() {
-  return <div />;
+const LIMIT = 12;
+
+export default function ListPage(props) {
+  const [productList, setProductList] = useState([]);
+  // const [buttonIndex, setButtonIndex] = useState(0);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  //data loading
+  useEffect(() => {
+    fetch(`http://localhost:3000/data/productListData.json`)
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data.product_groups);
+      });
+  }, []);
+
+  useEffect(() => {
+    const queryString = searchParams.toString('?');
+    fetch(`http://localhost:3000/data/productListData.json?${queryString}`)
+      .then(res => res.json())
+      .then(data => setProductList(data.product_groups));
+  }, [searchParams]);
+
+  const changePageByClick = index => {
+    const query = `?limit=${LIMIT}&offset=${index * LIMIT}`;
+    navigate(`/products${query}`);
+  };
+
+  //기능 수정 중 - 추가 구현 예정
+  // const selectCurrentButton = idx => {
+  //   Number(setButtonIndex(idx));
+  // };
+
+  // const goToPrev = index => {
+  //   index = 1 ? changePageByClick(0) : changePageByClick(index - 1);
+  // };
+
+  // const goToNext = index => {
+  //   index = 2 ? changePageByClick(1) : changePageByClick(index + 1);
+  // };
+
+  return (
+    <ListWrapper>
+      <ListHeader>
+        <CategoryTitle>
+          아웃도어 <BiChevronDown />
+        </CategoryTitle>
+        <ListMenu />
+        <SubMenus />
+      </ListHeader>
+      <ProductListTitle>
+        인기있는 아웃도어 <TotalList>{productList.length}</TotalList>
+      </ProductListTitle>
+
+      <ProductList>
+        {productList.map(product => {
+          return (
+            <Product
+              key={product.id}
+              address={product.address}
+              image={product.image_url}
+              title={product.product_name}
+              price={product.price}
+              discount={product.discounted_price}
+              star={product.star_point}
+            />
+          );
+        })}
+      </ProductList>
+      <PageWrapper>
+        <PageButton>&lt;</PageButton>
+        <PageNumber
+          onClick={() => {
+            changePageByClick(0);
+          }}
+        >
+          1
+        </PageNumber>
+        <PageNumber
+          onClick={() => {
+            changePageByClick(1);
+          }}
+        >
+          2
+        </PageNumber>
+        <PageNumber
+          onClick={() => {
+            changePageByClick(2);
+          }}
```

Comment:
```suggestion
          onClick={() => changePageByClick(2)}
```
한줄로 쓸 수 있으면 한줄로 써주는게 좋아보입니다